src/internt-vedtak/VurderingerMålgruppe.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { MålgruppeFaktaOgVurderingerDto, Vilkårperiode } from './typer/vilkårperiode';
+import { Vurdering } from './Vurdering';
+
+export const VurderingerMålgruppe: React.FC<{ målgruppe: Vilkårperiode }> = ({ målgruppe }) => {
+    const { medlemskap, utgifterDekketAvAnnetRegelverk } =
+        målgruppe.faktaOgVurderinger as MålgruppeFaktaOgVurderingerDto;
```

Comment:
Ettersom ikke målgruppe brukes til noe annet enn å hente ut faktaOgVurderinger, burde man kun sende med `MålgruppeFaktaOgVurderingerDto` for å unngå typcasting her?

Reply:
Fikset!